[CS2113-T11-1] Wellness360

genexus85 opened the pull request:
Wellness360 is a wellness app that is meant for stressed Engineering Students who prefer CLI over GUI and want to keep track of their overall wellbeing. Our app offers comprehensive tracking and management tools, providing personalized support to alleviate stress and enhance overall well-being.

Daviancold committed:
Add Class Diagram for current features of Fitness Motivator

Daviancold committed:
Add sequence diagram for current fitness motivator features

Daviancold committed:
Add class and sequence diagram to developer guide for fitness motivator

Includes description for both diagrams. Accurate with current features,
may change with future new features

OKW32 committed:
Merge pull request #113 from Daviancold/branch-ExerciseMotivator

Add Developer Guide for Fitness Motivator

Daviancold committed:
Add draft code for new fitness goal feature

Note that certain features may be broken.

genexus85 committed:
Edit developer guide for habit tracker

genexus85 committed:
Merge branch 'master' into branch-DeveloperGuide

damiwee committed:
Merge pull request #114 from genexus85/branch-DeveloperGuide

Edit developer guide for habit tracker

Daviancold committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-T11-1/tp into branch-ExerciseMotivator

damiwee committed:
Add junit tests for reflection feature

damiwee committed:
Fix coding style violation

damiwee committed:
Add comments to junit tests

Daviancold committed:
Merge pull request #116 from damiwee/branch-ReflectJunitV2.0

Update junit tests for reflection feature

Daviancold committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-T11-1/tp into branch-ExerciseMotivator

Daviancold committed:
Add goal feature for Fitness Motivator

Daviancold committed:
Add Java Docs and Comments for fitness goal feature

Daviancold committed:
Fix Build Gradle Issue for Fitness Motivator

Daviancold committed:
Remove data file for Fitness Motivator

Daviancold committed:
Change message variable to constants in FitnessMotivator Class

damiwee committed:
Add junit tests for reflection commands

Let's
* Focus on testing constructors and checking of exit flags
* Exclude calling on execution as it will become a
repetition of testing methods in reflection manager class

Daviancold committed:
Refactor code for Fitness Motivator

Constant Strings were extracted into constant classes

Daviancold committed:
Fix issue with current JUnit Tests for Fitness Motivator

damiwee committed:
Merge pull request #117 from Daviancold/branch-ExerciseMotivator

Add Goals Feature for Fitness Motivator

Daviancold committed:
Add fitness delete feature for Fitness Motivator

Deletes an exercise from the list of exercises stored in the Fitness
Motivator

Daviancold committed:
Fix coding convention in FitnessCommandParser class

Daviancold committed:
Remove unused import in main

Daviancold committed:
Add Java Docs for fitness delete feature

Daviancold committed:
Minor changes to comment and exception handling in command parser

Daviancold committed:
Fix CheckStyle Violations

OKW32 committed:
Merge pull request #119 from Daviancold/branch-ExerciseMotivator

Add fitness delete feature for Fitness Motivator

Daviancold committed:
Merge pull request #1 from Daviancold/branch-ExerciseMotivator

Merge Branch-ExerciseMotivator

Daviancold committed:
Merge pull request #118 from damiwee/branch-ReflectJunitV2.0

Add junit tests for reflection commands

Daviancold committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-T11-1/tp

Daviancold committed:
Refactor code to reduce arrowhead in ExerciseGoalList Class

Daviancold committed:
Add draft for Ui Component in DeveloperGuide

Daviancold committed:
Update user guide for new features in FitnessMotivator

genexus85 committed:
Merge pull request #125 from Daviancold/master

Refactor code to reduce Arrowhead in ExerciseGoalList Class

Daviancold committed:
Add minor changes to the User Guide

genexus85 committed:
Merge pull request #127 from Daviancold/branch-UserGuide

Update User Guide for new features in FitnessMotivator

Daviancold committed:
Update Class Diagram and PlantUML for FitnessMotivator

Daviancold committed:
Update Developer Guide for new features in FitnessMotivator

Moved ExerciseType Enum out of the exercise Package into the main
fitness package as well.

JingHaoooo committed:
Add junit tests for focus feature

JingHaoooo committed:
Refactor code to improve coding standards

Daviancold committed:
Merge pull request #129 from JingHaoooo/branch-FocusTest

update junit tests for focus feature

JingHaoooo committed:
Merge pull request #128 from Daviancold/branch-DeveloperGuide

Update Class Diagram and Developer Guide for FitnessMotivator

damiwee committed:
Merge branch 'master' into branch-DeveloperGuideV2.0

Daviancold committed:
Add Ui Component into Developer Guide

damiwee committed:
Update user and developer guide

damiwee committed:
Update sequence diagram

damiwee committed:
Merge pull request #130 from Daviancold/branch-DeveloperGuide

Add Ui Component into Developer Guide

Daviancold committed:
Merge pull request #131 from damiwee/branch-DeveloperGuideV2.0

Update user and developer guides

Daviancold committed:
Edit Fitness Class Diagram

Daviancold committed:
Add v2.0 user stories for Fitness Motivator

Daviancold committed:
edit parser for fitness command

Made fitness command non-case sensitive

Daviancold committed:
Add Manual Testing test case for Fitness Motivator in Dev Guide

Daviancold committed:
Update Fitness Motivator command parsers to increase defensiveness

genexus85 committed:
Add JUnit tests for habit tracker and habit classes

genexus85 committed:
Merge branch 'master' into branch-JUnit

Daviancold committed:
Merge pull request #134 from genexus85/branch-JUnit

Update JUnit tests for habit tracker feature

Daviancold committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-T11-1/tp into branch-DeveloperGuide

Daviancold committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-T11-1/tp into branch-ExerciseMotivator

damiwee committed:
Merge pull request #133 from Daviancold/branch-ExerciseMotivator

Minor changes to FitnessMotivator Feature

OKW32 committed:
Update developer guide for Storage

damiwee committed:
Merge pull request #132 from Daviancold/branch-DeveloperGuide

Minor Changes to Developer Guide for Fitness Motivator

Daviancold committed:
Remove input sensitivity for Wellness360

OKW32 committed:
Reduce arrowhead style coding in SleepCycleList class

damiwee committed:
Fix spacing in developer guide for reflection manual testing

damiwee committed:
Merge pull request #135 from Daviancold/branch-Main

Remove input sensitivity for Wellness360

OKW32 committed:
Remove input-output test

OKW32 committed:
Merge pull request #137 from damiwee/branch-DeveloperGuideV2.0

Fix spacing in developer guide for reflection manual testing

damiwee committed:
Merge pull request #136 from OKW32/branch-SleepTrackerV2

Reduce arrowhead style coding in SleepCycleList class

damiwee committed:
Merge branch 'master' into branch-UserGuideV2.0

Daviancold committed:
Add JUnit test for Fitness Goals Feature in Fitness Motivator

damiwee committed:
Merge pull request #138 from OKW32/branch-DeveloperGuide

Update developer guide for Storage

OKW32 committed:
Merge pull request #139 from Daviancold/branch-JUnitTest

Add JUnit test for Fitness Goals Feature in Fitness Motivator

Daviancold committed:
Minor changes to User Guide

Made minor changes to developer guide as well

OKW32 committed:
Add minor changes to guides for SleepTracker

JingHaoooo committed:
Bug fix for countdown timer when using command exit.

Daviancold committed:
Minor Changes to Command Cheatsheet in User Guide

JingHaoooo committed:
Merge pull request #140 from OKW32/branch-Guides

Add minor changes to guides for SleepTracker

damiwee committed:
Update user guide

Daviancold committed:
Merge pull request #143 from damiwee/branch-UserGuideV2.0

Update user guide

JingHaoooo committed:
Refactor code to improve coding quality

JingHaoooo committed:
Refactor codee to comply with coding quality

Daviancold committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-T11-1/tp into branch-UserGuide

JingHaoooo committed:
Merge pull request #141 from Daviancold/branch-UserGuide

Minor changes to User Guide

genexus85 committed:
Update habit class and sequence diagrams

genexus85 committed:
Update developer guide for habit tracker

genexus85 committed:
Refactor code for habit tracker class

genexus85 committed:
Update user guide for habit tracker

genexus85 committed:
Merge branch 'master' into branch-DeveloperGuide

JingHaoooo committed:
Update Focus timer user guide

JingHaoooo committed:
Merge pull request #144 from genexus85/branch-DeveloperGuide

Update developer and user guide for habit tracker

Daviancold committed:
Merge pull request #142 from JingHaoooo/branch-FocusV2

Bug fix for countdown timer when using command exit.

genexus85 committed:
Merge pull request #145 from JingHaoooo/branch_FocusUserGuide

Update Focus timer user guide

genexus85 committed:
Refactor set priority command class to abstract out complex conditions

damiwee committed:
Merge branch 'master' into branch-ReflectionV2.0

JingHaoooo committed:
Update user guide for focus timer

damiwee committed:
Merge pull request #146 from genexus85/branch-Habit-v2.0

Refactor set priority command class

damiwee committed:
Merge branch 'master' into branch-ReflectionV2.0

OKW32 committed:
Add Junit tests for Sleep Tracker

JingHaoooo committed:
Merge pull request #148 from OKW32/branch-Guides

Add Junit tests for Sleep Tracker

OKW32 committed:
Merge pull request #147 from JingHaoooo/branch_FocusUserGuide

Add additional details on user guide for focus timer

genexus85 committed:
Update user guide for habit tracker

damiwee committed:
Merge branch 'master' into branch-UserGuideV2.0

damiwee committed:
Merge branch 'master' into branch-ReflectionV2.0

Daviancold committed:
Fix bug for delete and help command in Fitness Motivator

Daviancold committed:
Merge pull request #149 from genexus85/branch-UserGuide

Update user guide for habit tracker

damiwee committed:
Add check for duplicate favorite questions saved

damiwee committed:
Update user guide

genexus85 committed:
Merge pull request #150 from Daviancold/branch-ExerciseMotivator

Fix bug for delete and help command in Fitness Motivator

genexus85 committed:
Merge pull request #152 from damiwee/branch-UserGuideV2.0

Update user guide for reflection

genexus85 committed:
Merge pull request #151 from damiwee/branch-ReflectionV2.0

Add check for duplicate saved question

BryanCastorius commented on docs/diagrams/storage/StorageSequenceDiagramSleep.png:
Shouldn't all the activation bars be enclosed (e.g. some activation bars, top and bottom parts are not closed by a line)?

BryanCastorius commented on docs/diagrams/storage/StorageSequenceDiagramSleep.png:
Aren't all classes supposed to start with a semicolon (e.g. :SleepTrackerStorage)?

BryanCastorius commented on docs/diagrams/focus/FocusSequenceDiagram.png:
For the activation bar underneath the :PlaceholderFocusCommand class, there's a deletion (X), however after the (X), is the lifeline supposed to still be continued?

tinaliu27 commented on docs/diagrams/sleep/Sleep.puml:
```diff
@@ -0,0 +1,94 @@
+
+SleepCycleList *-- "*" SleepCycle : contains
+
+@enduml
```
For this class diagram, it was a bit confusing trying to following the creates inheritance from SleepCommandParser and Command. Perhaps try stretching out the diagram to make the lines more clear?

tinaliu27 replied:
Otherwise great job with the diagram! Very detailed :)

BryanCastorius commented on docs/diagrams/storage/StorageSequenceSleep.png:
When invoking a method, we do not need to write in "calls..." right?

tinaliu27 commented on docs/diagrams/focus/FocusStateDiagram.puml:
```diff
@@ -0,0 +1,17 @@
+StateTimerRunning --> StateTimerRunning: check
+
+
+@enduml
```
Great job with the diagrams! Just make sure that when you are using <<class>> you wanna use a lower case!

BryanCastorius reviewed (commented):
Overall, a complete class and sequence diagrams have been provided with comprehensive explanations. Just a few minor convention error here and there on the sequence diagram.

cheeseong2001 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+* The labels of the arrows represents the commands.
+
+#### Focus Sequence Diagram
+![FocusSequenceDiagram.png](diagrams/focus/FocusSequenceDiagram.png)
```
For the sequence diagram step 7, maybe it be consistent with other sequence diagrams step 7?

cheeseong2001 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+#### Implementation
+
+##### Class Diagram
+![SleepDiagram.png](diagrams/sleep/SleepDiagram.png)
```
Is the this class diagram too messy?

ParthGandhiNUS replied:
While this diagram is very comprehensive, perhaps can consider breaking it up into more manageable smaller diagrams. Having so many lines which are overlapping can be confusing to developers.

cheeseong2001 commented on docs/DeveloperGuide.md:
Overall good job on the very descriptive developer guide. But maybe consider that it might be too wordy?

cheeseong2001 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+#### Class Diagram
+<!-- Insert image and description of each class, with its overview, attributes,
+methods, dependencies and UML Notes -->
+![FitnessClassDiagram](./diagrams/fitness/FitnessClassDiagram.png)
```
Is this too small for viewing?

hafizuddin-a replied:
Detailed diagram. Might want to consider resizing and insert newline after the diagram

cheeseong2001 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+<!-- Insert image and description of each class, with its overview, attributes,
+methods, dependencies and UML Notes -->
+![FitnessClassDiagram](./diagrams/fitness/FitnessClassDiagram.png)
+Note that certain details described below have been omitted from Class Diagram for simplicity and to improve readability.
```
Is there suppose to be a newline after the diagram? There is minor formatting issue

cheeseong2001 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+
+#### Class Diagram
+<!-- Insert Class Diagram here -->
+![UiClassDiagram.png](diagrams/Ui/UiClassDiagram.png)
```
Is the coloured "C" icon supposed to be present?

rismm commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+
+### UI component
+#### Description
+`Ui` component is in charge of reading in user input and printing output. </br> </br>
```
It appears that the line break in this line is in the wrong syntax? Maybe use `<br>` instead or remove the line break altogether.

rismm commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+#### Implementation
+
+##### Class Diagram
+![StorageClassDiagram.png](diagrams/storage/StorageClassDiagram.png)
```
Do consider adding `:void` to void methods in the class diagram for better clarity

ParthGandhiNUS replied:
Class diagram looks neat and comprehensive! Good Job!

hafizuddin-a replied:
Is the coloured C supposed to be present

rismm commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+#### Implementation
+
+##### Class Diagram
+![ReflectionClassDiagram.png](diagrams/reflection/ReflectionClassDiagram.png)
```
Might want to simplify or make the diagram's font larger. The class diagram is a little hard to see and is quite convoluted

rismm commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+      * `questionBank`: Instance of `ReflectionQuestionBank` managing reflection questions.
+      * `favoriteReflectionsList`: Instance of `FavoriteReflectionsList` managing favorite reflection questions.
+    * Methods:
+      * `printFiveRandomQuestions()`: Prints five random reflection questions.
+      * `saveReflectionQuestion(int reflectionId)`: Saves a reflection question to favorites based on ID.
+      * `unsaveReflectionQuestion(int reflectionId)`: Removes a reflection question from favorites based on ID.
+      * `printFavourites()`: Prints the list of favorite reflection questions.
+      * `printHelpMenu()`: Prints the help menu with available reflection commands.
```
Maybe avoid explaining methods where the method names are quite self-explanatory to avoid over explanation.

rismm commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+##### Sequence Diagram 
+
+##### Loading Data Sequence Diagram
+![StorageSequenceDiagramSleep.png](diagrams/storage/StorageSequenceDiagramSleep.png)
+
+* Note: HabitTrackerStorage works in a similar way as SleepTrackerStorage
+
+When `Main` starts, `SleepTracker` object is created. `SleepTrackerStorage loadSleepListFromFile()` method is called to 
+Begin loading sleep cycles from text file. `SleepCycleList` is returned to `SleepTracker` class and Sleep Tracker class 
+is successfully instantiated.
+
+##### Saving Data Sequence Diagram
+![StorageSequenceSleep.png](diagrams/storage/StorageSequenceSleep.png)
```
Consider hiding the footbox of classes at the bottom of the sequence diagram

rismm commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+#### Implementation
+
+##### Class Diagram
+![HabitClassDiagram.png](diagrams/habit/HabitClassDiagram.png)
```
Could consider excluding some methods and parameters to simplify the diagram as it looks too bulky.

rismm commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+    * Attributes:
+        * `habitList`: Private attribute representing the list of habits. It is an ArrayList of Habit objects.
+    * Methods:
+        * `HabitTracker()`: Constructs a HabitTracker object. 
+        * `getNumberOfHabits()`: Returns the number of habits in the habitList.
+        * `addHabit(Habit newHabit)`: Adds a new Habit object into the habitList.
+        * `listHabits()`: Prints a list of all habits in habitList .
+        * `isValidHabitID(int habitID)`: Check if a habit ID is valid by comparing with the size of habitList.
+        * `updateHabitCount(int habitID, String updatedCount)`: Update the habit count of a habit.
+        * `deleteHabit(int habitID)`: Delete a habit from habitList.
+        * `setPriorityLevel(int habitID, String priority)`: Set the priority of a habit.
+        * `sortHabits()`: Sort the habits in habitList according to the habits' priorities.
+        * `clearHabitList()`: Delete all habits from habitList.
```
Could consider leaving out listing methods where the method names are quite self-explanatory

rismm reviewed (commented):
Overall good job on the DG! Do consider simplifying the DG as it seems a little too overwhelming.

ParthGandhiNUS commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ... | I want to ...                                                                         | So that I can ...                                                   |
```
Comprehensive User stories being used. Good Job!! Warms the cockles in my heart!

hafizuddin-a commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+
+
+#### Sequence Diagram
+![HabitSequenceDiagram.png](diagrams/habit/HabitSequenceDiagram.png)
```
Perhaps the words on this diagram may be hard to read

hafizuddin-a commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,1282 @@
+    - `printMessagewithSepNewLine(String message)`: A public static method used to print a message with a separation line.
+    - `printMessageWithoutNewLine(String message)`: A public static method used to print a message without a separation line.
+    - `printList(ArrayList list, String message)`: A public static method to print a list of type `ArrayList`, along with a message and a separation line.
+### Command parser component
```
Perhaps accidental empty section

hafizuddin-a reviewed (commented):
Overall very detailed DG. Good effort

damiwee committed:
Fix developer guide documentation bug

Let's
* hide footboxes for sequence diagrams
* remove unnecessary html syntax
* increase font size for class diagrams

damiwee committed:
Fix letter casing for 'get' word in user stories

damiwee committed:
Update user guide FAQ on editing text files

damiwee committed:
Add missing documentation for exit command

damiwee committed:
Merge pull request #182 from damiwee/branch-BugFixesV2.1

Documentation bug fixes

JingHaoooo committed:
Update User Guide for focus timer

JingHaoooo committed:
Improve readability of user guide for focus timer

genexus85 committed:
Update habit class and sequence diagrams

damiwee committed:
Merge pull request #183 from JingHaoooo/branch-FocusV2.1

Update User guide for focus timer

genexus85 committed:
Update user guide for habit tracker

genexus85 committed:
Update developer guide for habit tracker

genexus85 committed:
Refactor code for habit tracker to meet coding standards

genexus85 committed:
Merge branch 'master' into branch-DeveloperGuide

JingHaoooo committed:
Bug fix: Refactor error messages

OKW32 committed:
Merge pull request #184 from genexus85/branch-DeveloperGuide

Documentation bug fixes and additions for habit tracker

JingHaoooo committed:
Update Developer guide to include test cases for focus feature.

JingHaoooo committed:
Update test cases in developer guide for focus feature

JingHaoooo committed:
Refactor developer guide for focus feature

genexus85 committed:
Merge pull request #185 from JingHaoooo/branch-FocusV2.1

Bug fixes and update developer guide for focus feature

OKW32 committed:
Fix bug for add sleep cycle command

Negative hours and hours greater than 24 could be added to the sleep
cycle list.

Let's add additional exception handlers to prevent the addition of
such values into the list.

OKW32 committed:
Add feature to auto delete older sleep cycles

Let's add checks to see if the total number of hours tracked by
Sleep Tracker is about to exceed MAX_VALUE of double, if so, delete
older sleep cycles.

OKW32 committed:
Add Sleep Help feature

genexus85 committed:
Merge pull request #186 from OKW32/branch-SleepTrackerV2.1

Bug fix for Sleep Tracker

OKW32 committed:
Update guides for Sleep Tracker to include help menu

Daviancold committed:
Bug Fix for add exercise command in Exercise Motivator

Fix bug where input is not allowed if delimiter is used in any of the
parameters

Daviancold committed:
Fix bug in delete command for Fitness Motivator

Fix issue where program crashes if an out of bounds index is given in
the command

genexus85 committed:
Merge pull request #187 from OKW32/branch-Guides

Update guides for Sleep Tracker

Daviancold committed:
Fix issues with build.gradle

Daviancold committed:
Fix new issues with Build.Gradle

Daviancold committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-T11-1/tp into branch-UserGuide

Daviancold committed:
Edit User Guide for Fitness Motivator Bug Fix

genexus85 committed:
Merge pull request #188 from Daviancold/branch-ExerciseMotivator

Fix issues from PE-D for Fitness Motivator

genexus85 committed:
Merge pull request #189 from Daviancold/branch-UserGuide

Edit User Guide for Bug Fix

Daviancold committed:
Update Fitness And UI Sequence Diagrams

Daviancold committed:
Update Developer guide with bug fix changes

Daviancold committed:
Minor Update to sequence Diagram for Developer Guide

genexus85 committed:
Merge pull request #190 from Daviancold/branch-DeveloperGuide

Update Developer Guide

damiwee committed:
Merge branch 'master' into branch-DeveloperGuideV2.1

OKW32 committed:
Update Designer Guide for Storage class

OKW32 committed:
Upload PPP for Ong Kan Wu

damiwee committed:
Add overall architecture image and description

OKW32 committed:
Merge pull request #192 from damiwee/branch-DeveloperGuideV2.1

Add overall architecture description and image

damiwee committed:
Merge pull request #191 from OKW32/branch-Guides

Update Designer Guide for Storage class

OKW32 committed:
Add hyperlinks in PPP for Ong Kan Wu

OKW32 committed:
Add hyperlink in README.md to PPP

JingHaoooo committed:
Merge pull request #193 from OKW32/branch-Guides

Add PPP for Ong Kan Wu

damiwee committed:
Add PPP for damiwee

OKW32 committed:
Merge pull request #194 from damiwee/branch-PPPV2.1

Add PPP for damiwee

OKW32 committed:
Edit PPP for Ong Kan Wu

OKW32 committed:
Edit documentation bug in PPP for Ong Kan Wu

damiwee committed:
Merge pull request #195 from OKW32/branch-Guides

Edit PPP for Ong Kan Wu

OKW32 committed:
Edit User Guide for Starting Wellness360

JingHaoooo committed:
Add help command for focus timer

JingHaoooo committed:
Update user guide for focus timer

damiwee committed:
Merge pull request #196 from OKW32/branch-Guides

Edit User Guide for Starting Wellness360

JingHaoooo committed:
Change count down timer warning to 3 seconds

JingHaoooo committed:
Refactor code to improve coding quality

genexus85 committed:
Merge pull request #197 from JingHaoooo/branch-FocusV2.1

Add help command for focus timer and reduce countdown timer warning

OKW32 committed:
Remove unnecessary spacing in designer guide for Sleep Tracker

JingHaoooo committed:
Update developer guide for focus timer

JingHaoooo committed:
Merge branch 'master' into branch-FocusV2.1

JingHaoooo committed:
Merge pull request #198 from OKW32/branch-Guides

Remove unnecessary spacing in designer guide for Sleep Tracker

JingHaoooo committed:
Update developer guide to add missing hyperlinks

JingHaoooo committed:
Add PPP for Lim Jing Hao

damiwee committed:
Merge pull request #199 from JingHaoooo/branch-FocusV2.1

Add PPP for Lim Jing Hao and update developer guide for focus timer

Daviancold committed:
Minor Changes to Ui Message Display

JingHaoooo committed:
Merge pull request #200 from Daviancold/branch-ExerciseMotivator

Minor Changes to Ui Message Display

genexus85 committed:
Add command parser class and sequence diagrams

genexus85 committed:
Update command parser developer guide

Daviancold committed:
Merge pull request #201 from genexus85/branch-DeveloperGuide

Update developer guide for command parser component

genexus85 committed:
Add PPP for Ari

OKW32 committed:
Edit sequence diagram for Storage component

damiwee committed:
Merge pull request #202 from genexus85/branch-DeveloperGuide

Add PPP for Ari

damiwee committed:
Merge pull request #203 from OKW32/branch-Guides

Edit sequence diagram for Storage component

JingHaoooo committed:
Fix visual bug for Count up timer

JingHaoooo committed:
Update README

damiwee committed:
Merge pull request #204 from JingHaoooo/branch-FocusV2.1

Update readme

damiwee committed:
Merge branch 'master' into branch-DeveloperGuideV2.1

damiwee committed:
Update guides

damiwee committed:
Merge pull request #205 from damiwee/branch-DeveloperGuideV2.1

Update guides

Daviancold committed:
Add PPP for Davian Kho

genexus85 committed:
Merge pull request #206 from Daviancold/branch-DeveloperGuide

Add PPP for Davian Kho

JingHaoooo committed:
Update readme and refactor PPP file name

JingHaoooo committed:
Merge pull request #207 from JingHaoooo/branch-FocusV2.1

Update readme and refactor JingHao PPP file name

JingHaoooo committed:
Update developer guide for focus timer

JingHaoooo committed:
Update hyperlink for Jing Hao PPP

JingHaoooo committed:
Merge pull request #208 from JingHaoooo/branch-FocusV2.1

Update Jing Hao PPP and focus timer developer guide

damiwee committed:
Merge branch 'master' into branch-DeveloperGuideV2.1

damiwee committed:
Update hyperlinks

damiwee committed:
Merge pull request #209 from damiwee/branch-DeveloperGuideV2.1

Update hyperlinks

JingHaoooo committed:
Remove old FocusTimerTest file

JingHaoooo committed:
Merge pull request #210 from JingHaoooo/branch-FocusV2.1

Remove old FocusTimerTest file

JingHaoooo committed:
Update user guide command summary

JingHaoooo committed:
Bug fix for focus timer

JingHaoooo committed:
Merge pull request #211 from JingHaoooo/branch-FocusV2.1

Bug fixes for focus timer and update user guide

JingHaoooo committed:
Bug fix for focus timer feature in user guide

JingHaoooo committed:
Update user guide command summary

JingHaoooo committed:
Merge pull request #212 from JingHaoooo/branch-FocusV2.1

Bug fix for focus timer feature in user guide

genexus85 committed:
Update greet user message

genexus85 committed:
Add new line to IO test

OKW32 committed:
Edit file name for Ong Kan Wu PPP

genexus85 committed:
Edit greet message

genexus85 committed:
Merge pull request #213 from genexus85/branch-Habit-v2.1

Update greet user message

Daviancold committed:
Edit formatting in Developer Guide

Daviancold committed:
Merge pull request #215 from Daviancold/branch-DeveloperGuide

Edit formatting in Developer Guide

OKW32 committed:
Edit header for Ong Kan Wu's PPP

genexus85 committed:
Merge pull request #214 from OKW32/branch-Guides

Edit file name for Ong Kan Wu PPP

genexus85 committed:
Update UG for focus feature

genexus85 committed:
Merge pull request #216 from genexus85/branch-DeveloperGuide

Update UG for focus feature

JingHaoooo committed:
Refactor developer guide to improve fomatting

JingHaoooo committed:
Merge pull request #217 from JingHaoooo/branch-FocusV2.1

Refactor developer guide to improve fomatting

JingHaoooo committed:
Bug fix for developer guide

JingHaoooo committed:
Merge pull request #218 from JingHaoooo/branch-FocusV2.1

Bug fix for developer guide

JingHaoooo committed:
Bug fix: Fix developer guide numbering

JingHaoooo committed:
Merge pull request #219 from JingHaoooo/branch-FocusV2.1

Bug fix: Fix developer guide numbering

JingHaoooo committed:
Fix bug for habit component in developer guide

JingHaoooo committed:
Fix bug for reflect component in developer guide

JingHaoooo committed:
Merge pull request #220 from JingHaoooo/branch-FocusV2.1

Fix bug for reflect component in developer guide

JingHaoooo committed:
Bug fix for focus timer component in developer guide

JingHaoooo committed:
Merge pull request #221 from JingHaoooo/branch-FocusV2.1

Bug fix for focus timer component in developer guide

genexus85 committed:
Add page breaks in DG

genexus85 committed:
Merge pull request #222 from genexus85/branch-DeveloperGuide

Add page breaks in DG

genexus85 committed:
Edit DG formatting

genexus85 committed:
Merge pull request #223 from genexus85/branch-DeveloperGuide

Edit DG formatting

genexus85 committed:
Add line spacing to DG

genexus85 committed:
Merge pull request #224 from genexus85/branch-DeveloperGuide

Add line spacing to DG

genexus85 committed:
Edit DG file formatting

genexus85 committed:
Merge pull request #225 from genexus85/branch-DeveloperGuide

Edit DG file formatting

damiwee committed:
Merge branch 'master' into branch-DeveloperGuideV2.1

damiwee committed:
Fix format for pdf conversion

damiwee committed:
Merge pull request #226 from damiwee/branch-DeveloperGuideV2.1

Merge branch 'master' into branch-DeveloperGuideV2.1

damiwee committed:
Fix pdf conversion format

damiwee committed:
Merge branch 'master' into branch-DeveloperGuideV2.1

damiwee committed:
Fix pdf conversion formatting

damiwee committed:
Merge pull request #227 from damiwee/branch-DeveloperGuideV2.1

Branch developer guide v2.1

damiwee committed:
Remove line breaks

damiwee committed:
Merge pull request #228 from damiwee/branch-DeveloperGuideV2.1

Remove line breaks
